qa/standalone/ceph-helpers.sh

```diff
@@ -152,11 +152,15 @@ function test_setup() {
 #
 function teardown() {
     local dir=$1
+    local dumplogs=$2
     kill_daemons $dir KILL
     if [ `uname` != FreeBSD ] \
         && [ $(stat -f -c '%T' .) == "btrfs" ]; then
         __teardown_btrfs $dir
     fi
+    if [ "$dumplogs" = "1" ]; then
+        display_logs $dir
+    fi
```

Comment:
@wjwithagen I'm just moving when this happens for now.  It would be nice if we could preserve the logs elsewhere.  Of course, it needs to handle teuthology vs local runs.